lib/diff/lcs/change.rb

```diff
@@ -28,7 +28,7 @@ def initialize(*args)
     unless Diff::LCS::Change.valid_action?(@action)
       raise "Invalid Change Action '#{@action}'"
     end
-    raise "Invalid Position Type" unless @position.kind_of? Fixnum
+    raise "Invalid Position Type" unless @position.kind_of? Integer
```

Comment:
Hello @halostatue, could you reply for above @koic comment?
I really hope that this issue is fixed, and bumped (released), because this is one of the reason that `cucumber` is not supported on Ruby 2.4.

Thank you.